docs/content/interop.md

```diff
@@ -25,83 +25,27 @@ You need to use the ICG to generate `.schema` files and type bindings for any ne
 ### How to use the ICG
 As the Interop Code Generator is an plugin for the Unreal toolbar, to use it you must first:
 *  build your game project to access the SpatialOS fork of Unreal Engine as well as the Unreal GDK. (You do this as part of the [installation and setup](../setup-and-installing.md), swapping the “Starter Project” and its file location for your game’s name and file location.)
-* set up a `DefaultEditorSpatialGDK.ini` file. See  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
+* List the classes that you want create typebindings for in the `SpatialGDKEditorSettings` from within the editor. See  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
 
```

Comment:
Suggest:
* list the classes that you want to create [typebindings](glossary.md#type-bindings) for in the in the `SpatialOS Unreal GDK` settings in the Unreal Engine 4 Editor. For instructions on how to do this, see the guidance on  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.

Why?
* Also, the style here is initial letter lower case. (It's moot in English whether it's upper case or lower case but as long as we are consistent, it's fine.)
* Below we say "`SpatialOS Unreal GDK` settings" so we should use the term consistently.